sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md

```diff
@@ -1812,6 +1813,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -StopNew2013Workflows
+Prevents creation of new SharePoint 2013 classic workflows.
```

Comment:
I would rephrase:

"Prevents creation of new SharePoint 2013 workflows utilizing the Workflow Manager service."

This will be more familiar to those who leveraged 2013 WFs in an on-prem environment. This is a suggestion, not a requirement -- approved as-is.